src/test/internal/render_with_styles.ts

```diff
@@ -18,13 +18,14 @@ import { createSerializer } from '@emotion/jest';
  * it's recommended that this util be used in the last test in the file.
  */
 export const renderWithStyles = (component: ReactElement) => {
-  expect.addSnapshotSerializer(
-    createSerializer({
-      classNameReplacer(className) {
-        return className;
-      },
-    })
-  );
-
-  return render(component);
+  afterAll(() => {
+    expect.addSnapshotSerializer(
+      createSerializer({
+        classNameReplacer(className) {
+          return className;
+        },
+      })
+    );
+    expect(render(component)).toMatchSnapshot('renderWithStyles');
```

Comment:
I'd keep this in English text for readability and to match the sentence structure of `it` blocks.

```suggestion
    expect(render(component)).toMatchSnapshot('renders with emotion styles');
```